server/modules/nodeDef/manager/nodeDefManager.js

```diff
@@ -15,14 +15,12 @@ const ActivityLogRepository = require('@server/modules/activityLog/repository/ac
 // ======= CREATE
 
 const insertNodeDef = async (user, surveyId, nodeDefParam, system = false, client = db) =>
-  await client.tx(async t => {
-    const [nodeDef] = await Promise.all([
+  await client.tx(async t => (await Promise.all([
       NodeDefRepository.insertNodeDef(surveyId, nodeDefParam, t),
       markSurveyDraft(surveyId, t),
       ActivityLogRepository.insert(user, surveyId, ActivityLog.type.nodeDefCreate, nodeDefParam, system, t)
-    ])
-    return nodeDef
-  })
+    ]))[0]
```

Comment:
it was more readable before

Reply:
right, I'll revert it to the previous version.